apps/api/src/index.ts

```diff
@@ -1,16 +1,28 @@
-import { createServer } from "./server.js";
+import { inspect } from "util";
 
-const port = process.env.PORT || 3000;
-const server = createServer();
+import { ILogger } from "@zkchainhub/shared";
 
-server
-    .get("/metrics/ecosystem", (_, res) => {
-        return res.json({ message: `/metrics/ecosystem` });
-    })
-    .get("/metrics/zkchain/:chainId", (req, res) => {
-        return res.json({ message: `/metrics/ecosystem/${req.params.chainId}` });
-    });
+import { App } from "./app.js";
+import { config } from "./common/config/index.js";
+import { Container } from "./container.js";
 
-server.listen(port, () => {
-    console.log(`api running on ${port}`);
+const container = new Container(config);
+const logger = container.get<ILogger>("ILogger");
```

Comment:
wdyt if we remove this class and do all the instantiations on the `main` function, i think it is more readable and better to understand

Reply:
I'd say that I agree.

Do you envision the `Container` class having another responsibilities apart from being used during the app initialization?